Makefile

```diff
@@ -101,7 +101,7 @@ develop: tags
 	$(PIP) install -e .[doc,test]
 
 test:
-	$(COVERAGE) run -m $(PYTEST) tests -v
+	$(COVERAGE) run -m $(PYTEST) tests -v -r sx
```

Comment:
I haven't tested what that option does, I just assumed it was a temporary debugging measure. But if it makes sense to keep it in permanently, then that's fine :-)